WcaOnRails/spec/features/competition_management_spec.rb

```diff
@@ -14,6 +14,22 @@
       click_button "Create Competition"
 
       expect(page).to have_text("Successfully created new competition!")
+
+      visit "/competitions/new"
+      click_button "Create Competition"
+
+      expect(page).to have_text("Name must end with a year")
```

Comment:
Very naive question here: shouldn't this be a separate scenario?
Like scenario 1 is creating a valid competition, scenario 2 is attempting to create an invalid competition?